web/srv/handlers.go

```diff
@@ -41,6 +41,13 @@ type (
 func (h *handler) handleIndex(w http.ResponseWriter, req *http.Request, p httprouter.Params) {
 	// when running the dashboard via `linkerd dashboard`, serve the index bundle at the right path
 	pathPfx := proxyPathRegexp.FindString(req.URL.Path)
+
+	TODO_CONFIGURABLE_PREFIX := "/linkerd"
```

Comment:
If you're up to doing the backend work, something like this would be required:

- In the [dashboard section of viz' values.yaml](https://github.com/linkerd/linkerd2/blob/main/viz/charts/linkerd-viz/values.yaml#L297) add a new entry `basePath`
- Have it passed as an [arg into the web container](https://github.com/linkerd/linkerd2/blob/main/viz/charts/linkerd-viz/templates/web.yaml#L74):
```yaml
        {{- if .Values.dashboard.basePath }}
        - -base-pathl={{.Values.dashboard.basePath }}
        {{- end}}
```
- Receive it as a new arg in [web/main.go](https://github.com/linkerd/linkerd2/blob/main/web/main.go#L29-L43):
```go
	basePath := cmd.String("base-pathl", "", "base path under which to serve the dashboard")
```
- Pass its value as a new arg to [srv.NewServer](https://github.com/linkerd/linkerd2/blob/main/web/main.go#L99)
- Add it to the [handler](https://github.com/linkerd/linkerd2/blob/main/web/srv/server.go#L116-L131)
- Finally, it should be accessible in `handleIndex()` as `h.basePath`.

Let me know how it goes! :-)